src/Sylius/Component/Core/Promotion/Action/UnitDiscountPromotionActionCommand.php

```diff
@@ -84,4 +98,13 @@ protected function createAdjustment(
 
         return $adjustment;
     }
+
+    private function calculate(int $itemTotal, ?int $minimumPrice, int $promotionAmount): int
```

Comment:
```suggestion
    private function calculate(int $unitTotal, ?int $minimumPrice, int $promotionAmount): int
```
using itemTotal here is a bit misleading